pkgs/build-support/build-fhs-userenv/chrootenv.c

```diff
@@ -170,7 +239,8 @@ int main(int argc, char *argv[]) {
   if (waitpid(cpid, &status, 0) < 0)
     errorf(EX_OSERR, "waitpid");
 
-  if (nftw(root, nftw_rm, getdtablesize(), FTW_DEPTH | FTW_MOUNT | FTW_PHYS) < 0)
+  if (nftw(root, nftw_rm, getdtablesize(), FTW_DEPTH | FTW_MOUNT | FTW_PHYS) <
+      0)
```

Comment:
Write return code to a local variable instead of line wrapping here for readability.

Reply:
Line wrapping will go away if I rename `nftw_rm`.